New Boxtribute theme

DurkoMatko opened the pull request:
change: styling to fit new Boxtribute theme

DurkoMatko commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Unify UI theming in preparation for rebranding #large](https://trello.com/c/on228c5S/476-unify-ui-theming-in-preparation-for-rebranding-large)

HaGuesto commented on assets/css/custom.css:
```diff
@@ -795,45 +804,34 @@ div.filter-applied {
 	padding-right: 16px;
-}
-
-.overview-filter-gender > div.btn.btn-sm.btn-default{
```
these css parts I added recently for the stock overview to connect the color of the levels with the color of the filters. I sther a reason why you removed it @DurkoMatko ?

aerinsol replied:
I asked him to, as it wasn't clear that's what it did.

HaGuesto replied:
So, can we add this again ?

HaGuesto replied:
@aerinsol

HaGuesto commented on assets/css/style.css:
```diff
@@ -431,7 +454,7 @@ body > iframe { display: none; }
 	background-image: -ms-linear-gradient(top,#f8f8f8,#f1f1f1);
 	border: 1px solid #c6c6c6;
-	color: #333;
+	/* color: #333; */
```
remove comment

DurkoMatko replied:
done

HaGuesto commented:
## more things I found
- The titles need more space to the menu
![image](https://user-images.githubusercontent.com/29065125/83868653-91d04800-a72b-11ea-9726-c4dc1b8bea54.png)

- the form field have round corners and old blue when clicking into them (focus)
![image](https://user-images.githubusercontent.com/29065125/83868904-f7243900-a72b-11ea-8f89-55da01818490.png)

- the tabs for forms look different than for lists
![image](https://user-images.githubusercontent.com/29065125/83868940-073c1880-a72c-11ea-8ede-43c09cdd818b.png)
![image](https://user-images.githubusercontent.com/29065125/83868972-1622cb00-a72c-11ea-98b0-811706475d50.png)

aerinsol commented:
@HaGuesto I don't agree. The header image is not suitable for login as it
serves a different purpose. Will add that and all other needed changes to
Trello card directly.

On Fri, Jun 5, 2020, 3:12 AM HaGuesto <notifications@github.com> wrote:

> *@HaGuesto* requested changes on this pull request.
> functional review
>
>    -
>
>    the login page should be adapted. I would do it like the proposed
>    facebook header image - red background + logo.
>    <https://boxwise.slack.com/files/U012Y0TPXP1/F0142QKN7K6/facebook_page_mockup_2020_psd_2b.jpg>
>    I added a checklist for this on the trello ticket.
>    -
>
>    There is an overflow of the ngo (base) name on mobile screens. There
>    is space on the right of the logo. Maybe the names can be moved more to the
>    left. Maybe we can decrease the font size and show the name of the ngo and
>    the base in two separate lines?
>    [image: image]
>    <https://user-images.githubusercontent.com/29065125/83862707-9ba17d80-a722-11ea-98be-721fa6a0ce2e.png>
>    -
>
>    @aerinsol <https://github.com/aerinsol> @DurkoMatko
>    <https://github.com/DurkoMatko> was it intended to replace the
>    sandwich menu with three dots? I find the three dots unclear and would
>    rather use the sandwich menu.
>    [image: image]
>    <https://user-images.githubusercontent.com/29065125/83862138-d48d2280-a721-11ea-839b-beeaca92fd91.png>
>    -
>
>    the search button in Boxes and beneficiaries lost its icon
>    [image: image]
>    <https://user-images.githubusercontent.com/29065125/83863371-8c6eff80-a723-11ea-8b92-b88a7d2a522a.png>
>    -
>
>    the behavior of filters in beneficiaries and boxes do not feel right.
>    the blue when hovering over them seems wrong. The edges are also round
>    where other buttons have a sharper edge.
>    [image: image]
>    <https://user-images.githubusercontent.com/29065125/83863660-ee2f6980-a723-11ea-9893-5637bbf710c4.png>
>    -
>
>    the left-side menu should not have a scrollbar for lef-right--> the
>    space for the menu should be increased.
>    [image: image]
>    <https://user-images.githubusercontent.com/29065125/83864073-87f71680-a724-11ea-82d1-7b3f9e13fcc2.png>
>    -
>
>    there is no way to get to the start page anymore. Before there was
>    always a link on the logo in the header menu to go back to the start page.
>    -
>
>    I would adjust the font-style for fancy-graphs and the start page.
>    They seem to large now in comparison to other text and I do not know about
>    the gray. @aerinsol <https://github.com/aerinsol>
>
> ------------------------------
>
> In assets/css/style.css
> <https://github.com/boxwise/dropapp/pull/364#discussion_r435815921>:
>
> > @@ -94,10 +103,10 @@ p{
>  	margin-top: -5px;
>  	margin-bottom: -5px;
>  }
> -.header-top-inner .nav>li>a:hover, .header-top-inner .nav>li>a:focus, .header-top-inner .nav .open>a, .header-top-inner .nav .open>a:hover, .header-top-inner .nav .open>a:focus{
> +/* .header-top-inner .nav .open>a:hover, .header-top-inner .nav .open>a:focus{
>
> please remove comment.
> ------------------------------
>
> In assets/css/custom.css
> <https://github.com/boxwise/dropapp/pull/364#discussion_r435820098>:
>
> >  	padding-right: 16px;
> -}
> -
> -.overview-filter-gender > div.btn.btn-sm.btn-default{
>
> these css parts I added recently for the stock overview to connect the
> color of the levels with the color of the filters. I sther a reason why you
> removed it @DurkoMatko <https://github.com/DurkoMatko> ?
> ------------------------------
>
> In assets/css/style.css
> <https://github.com/boxwise/dropapp/pull/364#discussion_r435824044>:
>
> > @@ -431,7 +454,7 @@ body > iframe { display: none; }
>  	background-image: -moz-linear-gradient(top,#f8f8f8,#f1f1f1);
>  	background-image: -ms-linear-gradient(top,#f8f8f8,#f1f1f1);
>  	border: 1px solid #c6c6c6;
> -	color: #333;
> +	/* color: #333; */
>
> remove comment
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/boxwise/dropapp/pull/364#pullrequestreview-425159624>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABP2BFH6Y7VQNGQ5BCNHBR3RVDAJ3ANCNFSM4NLCI6DQ>
> .
>

aerinsol commented:
Three dots is on purpose. Common to use this in modern design language.

On Fri, Jun 5, 2020, 1:02 PM Roanna Kong <roannapk@gmail.com> wrote:

> @HaGuesto I don't agree. The header image is not suitable for login as it
> serves a different purpose. Will add that and all other needed changes to
> Trello card directly.
>
> On Fri, Jun 5, 2020, 3:12 AM HaGuesto <notifications@github.com> wrote:
>
>> *@HaGuesto* requested changes on this pull request.
>> functional review
>>
>>    -
>>
>>    the login page should be adapted. I would do it like the proposed
>>    facebook header image - red background + logo.
>>    <https://boxwise.slack.com/files/U012Y0TPXP1/F0142QKN7K6/facebook_page_mockup_2020_psd_2b.jpg>
>>    I added a checklist for this on the trello ticket.
>>    -
>>
>>    There is an overflow of the ngo (base) name on mobile screens. There
>>    is space on the right of the logo. Maybe the names can be moved more to the
>>    left. Maybe we can decrease the font size and show the name of the ngo and
>>    the base in two separate lines?
>>    [image: image]
>>    <https://user-images.githubusercontent.com/29065125/83862707-9ba17d80-a722-11ea-98be-721fa6a0ce2e.png>
>>    -
>>
>>    @aerinsol <https://github.com/aerinsol> @DurkoMatko
>>    <https://github.com/DurkoMatko> was it intended to replace the
>>    sandwich menu with three dots? I find the three dots unclear and would
>>    rather use the sandwich menu.
>>    [image: image]
>>    <https://user-images.githubusercontent.com/29065125/83862138-d48d2280-a721-11ea-839b-beeaca92fd91.png>
>>    -
>>
>>    the search button in Boxes and beneficiaries lost its icon
>>    [image: image]
>>    <https://user-images.githubusercontent.com/29065125/83863371-8c6eff80-a723-11ea-8b92-b88a7d2a522a.png>
>>    -
>>
>>    the behavior of filters in beneficiaries and boxes do not feel right.
>>    the blue when hovering over them seems wrong. The edges are also round
>>    where other buttons have a sharper edge.
>>    [image: image]
>>    <https://user-images.githubusercontent.com/29065125/83863660-ee2f6980-a723-11ea-9893-5637bbf710c4.png>
>>    -
>>
>>    the left-side menu should not have a scrollbar for lef-right--> the
>>    space for the menu should be increased.
>>    [image: image]
>>    <https://user-images.githubusercontent.com/29065125/83864073-87f71680-a724-11ea-82d1-7b3f9e13fcc2.png>
>>    -
>>
>>    there is no way to get to the start page anymore. Before there was
>>    always a link on the logo in the header menu to go back to the start page.
>>    -
>>
>>    I would adjust the font-style for fancy-graphs and the start page.
>>    They seem to large now in comparison to other text and I do not know about
>>    the gray. @aerinsol <https://github.com/aerinsol>
>>
>> ------------------------------
>>
>> In assets/css/style.css
>> <https://github.com/boxwise/dropapp/pull/364#discussion_r435815921>:
>>
>> > @@ -94,10 +103,10 @@ p{
>>  	margin-top: -5px;
>>  	margin-bottom: -5px;
>>  }
>> -.header-top-inner .nav>li>a:hover, .header-top-inner .nav>li>a:focus, .header-top-inner .nav .open>a, .header-top-inner .nav .open>a:hover, .header-top-inner .nav .open>a:focus{
>> +/* .header-top-inner .nav .open>a:hover, .header-top-inner .nav .open>a:focus{
>>
>> please remove comment.
>> ------------------------------
>>
>> In assets/css/custom.css
>> <https://github.com/boxwise/dropapp/pull/364#discussion_r435820098>:
>>
>> >  	padding-right: 16px;
>> -}
>> -
>> -.overview-filter-gender > div.btn.btn-sm.btn-default{
>>
>> these css parts I added recently for the stock overview to connect the
>> color of the levels with the color of the filters. I sther a reason why you
>> removed it @DurkoMatko <https://github.com/DurkoMatko> ?
>> ------------------------------
>>
>> In assets/css/style.css
>> <https://github.com/boxwise/dropapp/pull/364#discussion_r435824044>:
>>
>> > @@ -431,7 +454,7 @@ body > iframe { display: none; }
>>  	background-image: -moz-linear-gradient(top,#f8f8f8,#f1f1f1);
>>  	background-image: -ms-linear-gradient(top,#f8f8f8,#f1f1f1);
>>  	border: 1px solid #c6c6c6;
>> -	color: #333;
>> +	/* color: #333; */
>>
>> remove comment
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/boxwise/dropapp/pull/364#pullrequestreview-425159624>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ABP2BFH6Y7VQNGQ5BCNHBR3RVDAJ3ANCNFSM4NLCI6DQ>
>> .
>>
>

DurkoMatko commented:
_I agree with this actually, but it's of lower importance since this is the "desktop" app and I don't want to waste @DurkoMatko's time on that. Rather than putting it on two separate lines, I would delete "BoxAid" and just show the base._
- deleted orgname

_the behavior of filters in beneficiaries and boxes do not feel right.
the blue when hovering over them seems wrong. The edges are also round
where other buttons have a sharper edge._
- removed onHover behavior

_the left-side menu should not have a scrollbar for lef-right--> the
space for the menu should be increased._
- can't reproduce

_ there is no way to get to the start page anymore. Before there was
always a link on the logo in the header menu to go back to the start page_
-done

DurkoMatko commented on assets/css/style.css:
```diff
@@ -94,10 +103,10 @@ p{
 	margin-bottom: -5px;
 }
-.header-top-inner .nav>li>a:hover, .header-top-inner .nav>li>a:focus, .header-top-inner .nav .open>a, .header-top-inner .nav .open>a:hover, .header-top-inner .nav .open>a:focus{
+/* .header-top-inner .nav .open>a:hover, .header-top-inner .nav .open>a:focus{
```
done

aerinsol commented:
> 
> _the form field have round corners and old blue when clicking into them (focus)_
> 
> * this was requested from Roanna (inputs on focus/active should have this blue outline)

I think it's the new light blue, not the old light blue.

> _the tabs for forms look different than for lists_
> 
> * this was the case in the old design as well. They are 2 completely different controls. What should I do? @aerinsol  :
>   ![image](https://user-images.githubusercontent.com/8964422/83950037-012b6200-a828-11ea-9c20-c1efc14caca2.png)
>   ![image](https://user-images.githubusercontent.com/8964422/83950029-ece76500-a827-11ea-83f9-af7232af198e.png)

@HaGuesto and @DurkoMatko leave it. We're not trying to massively improve the functionality of the old app at this point, just be consistent with the rebrand.

> _I would adjust the font-style for fancy-graphs and the start page. They seem to large now in comparison to other text and I do not know about the gray._
> 
> * Do u approve? @aerinsol
@DurkoMatko @HaGuesto can someone share me a screencap?

aerinsol commented:
I would say the following points.
- Ignore the header overflow. It's completely unimportant.
- leave the stock filter as is. Earlier behavior of color was not clear. If
someone complains, let me know and assign it to me.
- leave the color of text the start page.
- SECTION HEADERS should be black, not gray.


On Mon, Jun 8, 2020, 6:06 AM HaGuesto <notifications@github.com> wrote:

> *@HaGuesto* requested changes on this pull request.
>
> I will go through the list:
>
>    -
>
>    login page
>    looks good
>    -
>
>    mobile header overflow
>    @DurkoMatko <https://github.com/DurkoMatko> it is still overflowing
>    for longer base names and there would be space on the left of the base name.
>    [image: image]
>    <https://user-images.githubusercontent.com/29065125/84029504-c8f46280-a992-11ea-89e2-b09a3c5653d8.png>
>    -
>
>    Search button
>    @DurkoMatko <https://github.com/DurkoMatko> This one still does not
>    work. I looked in the code and you did not do any edits in the templates
>    there. Any idea why the search icon fa-search does not appear for
>    lists?
>    [image: image]
>    <https://user-images.githubusercontent.com/29065125/84032190-3dc99b80-a997-11ea-9b03-73347cb0abc2.png>
>    -
>
>    filter hover behavior
>    done. However, @DurkoMatko <https://github.com/DurkoMatko> if you
>    remove it from the filters, please also remove the hover behavior also from
>    the search input field.
>    -
>
>    link to start page
>    done
>    -
>
>    [] font style for fancygraphs and start page
>    @aerinsol <https://github.com/aerinsol> here the screenshot
>    [image: image]
>    <https://user-images.githubusercontent.com/29065125/84032818-2f2fb400-a998-11ea-80e5-d57fba4d9d6d.png>
>    -
>
>    [] focus color for input fields
>
> the form field have round corners and old blue when clicking into them
> (focus)
> I just checked the border color is still the old blue. #4cbac5
> @DurkoMatko <https://github.com/DurkoMatko> Please replace it.
>
>
>    - [] space between page title and header menu
>
> The titles need more space to the menu
> @DurkoMatko <https://github.com/DurkoMatko> and @aerinsol
> <https://github.com/aerinsol> I got no answer on this.
> @DurkoMatko <https://github.com/DurkoMatko> I believe that the new logo
> is taller than the old one. Maybe it works if you just decrease the B a bit.
> [image: image]
> <https://user-images.githubusercontent.com/29065125/84033643-77030b00-a999-11ea-9702-f14a71ff59e0.png>
>
> One more request for @DurkoMatko <https://github.com/DurkoMatko> - as a
> stand-in while we wait for the MVP of the mobile app to come out, can you
> unify the background and button style of mobile.php so it matches the new
> theme? Should be just a simple change to the background and button styles.
> @aerinsol <https://github.com/aerinsol> I would make a separate ticket
> for this and try to merge this asap. The PR already gets confusing with all
> the comments.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/boxwise/dropapp/pull/364#pullrequestreview-426214148>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABP2BFDSK5ZJ3RWBWYLLQ43RVTO6HANCNFSM4NLCI6DQ>
> .
>

HaGuesto commented:
Thanks @aerinsol!
I think the last steps for you @DurkoMatko are then:

- [ ]  Fix search button

- [ ] SECTION HEADERS should be black, not gray. (in the menu)

(- exchange the favicon if possible)

DurkoMatko commented:
@HaGuesto done those 3
